wondrous-app/components/Common/CornerWidget/index.tsx

```diff
@@ -0,0 +1,52 @@
+import Snackbar from '@mui/material/Snackbar';
+import dynamic from 'next/dynamic';
+import { createContext, useContext, useMemo, useState } from 'react';
+
+import { CornerWidgetType } from './types';
+
+const CornerWidgetComponent = dynamic(() => import('./Component'), { ssr: false, suspense: false });
+
+export const CornerWidgetContext = createContext(null);
+
+export const useCornerWidget = () => useContext(CornerWidgetContext);
+
+const CornerWidgetProvider = ({ children }) => {
+  const [value, setValue] = useState<CornerWidgetType>({
+    open: false,
+    id: null,
+    type: null,
+    orgName: null,
+    podName: null,
+  });
+  const { open, id, type, orgName, podName } = value;
+  const handleClose = () =>
+    setValue({
+      ...value,
+      open: false,
+    });
+  const cornerWidgetValue = useMemo(
```

Comment:
you should use `useMemo` hook for computed values, in your case, it's the same as 
```
const cornerWidgetValue = {
   setCornerWidgetValue: setValue,
}
```

Reply:
I'm following the eslint recommendations, https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-constructed-context-values.md. Should I ignore this?